test/integration/taxon_browsing_test.rb

```diff
@@ -216,9 +228,34 @@ def and_i_can_see_links_to_the_child_taxons_in_an_accordion
     end
   end
 
+  def and_i_can_see_the_overview_section_in_the_accordion
+    subsection = first('.subsection')
+
+    assert subsection.has_selector?('.subsection-title', text: /overview/i)
+    assert subsection.has_selector?(
+      '.subsection-content .subsection-list-item a',
+      count: 2
+    )
+  end
+
+  def then_there_is_no_overview_section_in_the_accordion
+    subsection = first('.subsection')
+
+    assert_equal(
+      false,
+      subsection.has_selector?('.subsection-title', text: /overview/i)
+    )
+  end
+
+  def and_there_is_no_content_presented_after_the_accordion
```

Comment:
The idea behind this step is so no one by mistake changes the conditionals in the taxon show page in a way that would show the tagged content underneath the accordion. The content was there before and this step was failing until I fixed those conditionals.

Does this make sense?